### PR TITLE
Separator component

### PR DESCRIPTION
**Pull request by nickgraz**
- Adjustable height that defaults to 32px

### CODE REVIEW

**Comment by nickgraz** [comment]
Current dependencies on/for this PR:
* **#472** <a href="https://app.graphite.dev/github/pr/terraware/web-components/472?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/web-components/472?utm_source=stack-comment).

**Comment by nickgraz** [comment]
publish rc

**Review by karthikbtf — APPROVED** [review]
Thanks so much!!! 
Is this worth a story just for coverage?

File: src/components/Separator/index.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import theme from '../../theme';
+
+const Separator = (props: { height?: string }) => {
```
Comment (karthikbtf) [inline src/components/Separator/index.tsx:5]: Oh, export this in the main index

**Comment by nickgraz** [comment]
> Thanks so much!!! Is this worth a story just for coverage?

Sorry, just seeing this now, will add a story

**Review by karthikbtf — APPROVED** [review]
Nice 🎉

**Pull request merged**